### ISSUE TITLE
How to exclude files?

### ISSUE THREAD

**Issue by trixlerjs** [m-a-r-c-e-l-i-n-o/nutra#2]
How can i exclude files from being tested? Is there any option for it? And if there is a option for it, does this also work for watch taks? Or isn't there any watch task yet?

**Comment by m-a-r-c-e-l-i-n-o** [comment]
> How can i exclude files from being tested?

Nutra uses the [node-glob](https://github.com/isaacs/node-glob) module to parse patterns, and you can exclude files with negation and more advanced techniques. For instance:
`src/{server/*,server/!(test|coverage)/**}.js`

which is similar to:
```
src/server/*.js
src/server/!(test|coverage)/**/*.js
```
will include all files in the `src/server/*.js` directory, but exclude the test and coverage folders.

If you have a use case where those techniques are not capable of resolving, please post it here. I'll be happy to consider an explicit exclusion feature.

**Comment by trixlerjs** [comment]
I can't remember exactly the issue atm, but sometimes with Istanbul it require you to include all files, but exclude some other files and the glob pattern couldn't be used. I faced this issue both with Mocha and Karma. I know Istanbul have it's own method for excluding files from being instrumented, but on windows - where I faced that issue - it didn't help.

And I had similiar issue with Angular 2 a couple of month ago, but a college fixed the issue so can't explain it better. I think it was related to `core.js`.
